Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -515,7 +515,7 @@
 "page-history-compare-accessibility-hint" = "Tap to select two revisions to compare";
 "page-history-compare-title" = "Compare";
 "page-history-minor-edits" = "minor edits";
-"page-history-revision-author-accessibility-label" = "Author: %@";
+"page-history-revision-author-accessibility-label" = "Author: $1";
```

Comment:
This and the feed card strings above were incorrect due to the lack of support for `%@` instead of `%1$@`